### DISCUSSION TITLE
Model is not loading on Microsoft  Hololens 2 headset

### DISCUSSION THREAD

**Discussion by NitinJ88** [google/model-viewer#4864] [Q&A]
### Description

We are trying to use AR with the Windows Hololens 2 headset but it is having problem. On model viewer AR button is visualized but when he clicks on it the background becomes visible and a loading animation appears but nothing appears.

#### Live Demo

https://glitch.com/edit/#!/model-viewer

### Version

- model-viewer: v3.2.1

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [*] WebXR
- [ ] SceneViewer
- [*] QuickLook

**Comment by elalish** [comment]
Indeed, our AR experience is only for Phone-based AR currently. There is interest in getting a headset UX built out as well, and even some good starting work on it on #4643. If you'd like to build that branch and give it a try and give some feedback, that would be more appreciated. PRs certainly welcome in this arena.
